printsorted/print_sorted.go

```diff
@@ -0,0 +1,26 @@
+package printsorted
+
+import (
+	"fmt"
+	"sort"
+)
+
+func PrintSorted(m map[int]string) {
+	for _, v := range sortMap(m) {
+		fmt.Print(v, " ")
+	}
+
+	fmt.Println()
+}
+
+func sortMap(m map[int]string) []string {
+	k, r := make([]int, 0, len(m)), make([]string, 0, len(m))
```

Comment:
For me sortMap, is not obvious name for function, because it's returns sorted by keys values not exactly map

Reply:
Fixed https://github.com/VSKut/go-temp/pull/6/commits/5d6aae4543833d019da853686f4654a1b4b2c954